src/Controller/UserController.php

```diff
@@ -65,4 +68,14 @@ public function edit(
             'form' => $form,
         ]);
     }
+
+    #[Route('/{id}', name: 'user_show', methods: ['GET', 'POST'])]
+    public function show(int $id, UserRepository $userrepository): Response
+    {
+        $user = $userrepository->findOneById($id);
+
+        return $this->renderForm('user/show.html.twig', [
```

Comment:
you don't pass a form to render, so you can go with render instead of renderForm